src/eko/runner/parts.py

```diff
@@ -90,7 +90,7 @@ def evolve_configs(eko: EKO) -> dict:
         n3lo_ad_variation=tcard.n3lo_ad_variation,
         matching_order=tcard.matching_order
         if tcard.matching_order is not None
-        else tcard.order,
+        else (tcard.order[0] - 1, tcard.order[1]),
```

Comment:
```suggestion
        else (tcard.order[0] - 1, tcard.order[1] - 1),
```
I know QED matchings are beyond our current scope, let's spend 3 characters now if ever we get there ...

Reply:
true and also in the legacy...

Reply:
in the old style `QED` was a boolean, which translated naturally into the new counting - in the new style we still need to translate - do you agree @niclaurenti ? (actually, are we putting this in the fits for the paper to 2?)

Reply:
Okay, then it should be fine.
But on the other hand we should add a minus sign here: 
https://github.com/NNPDF/eko/blob/10f0733635c012ce7e1f0fd4cdfc497619292830/src/eko/io/runcards.py#L216

Reply:
why? again in the old style PTO=0 means LO, no?

Reply:
no I meant on the the QED

Reply:
okay, yes - just remember to put also a `min` maybe

Reply:
> in the old style `QED` was a boolean, which translated naturally into the new counting - in the new style we still need to translate - do you agree @niclaurenti ? (actually, are we putting this in the fits for the paper to 2?)

In the old style `QED` was the order in $alpha$ of the splitting kernels, so if we had QED corrections to matching conditions it should have a `-1` as for the QCD order (In the fit I'm using `QED=2`).